Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -55,8 +54,9 @@ private string Me
 
         #region Grain overrides
 
-        public override Task OnActivateAsync()
+        public override async Task OnActivateAsync()
         {
+            await ReadStateAsync();
```

Comment:
Yes, the runtime will have already called Read before it calls OnActivateAsync.
That is one of the fundamental guarantees of the declarative persistence system.